internal/kms/query.go

```diff
@@ -0,0 +1,74 @@
+package kms
+
+const (
+	// oldestPendingOrRunningRun lists the currently running or oldest
+	// run that has not yet completed for the specified table name.
+	oldestPendingOrRunningRun = `
```

Comment:
Which part is confusing? I tried looking at a few other `query.go`'s and they do a similar sort of indentation (with not discernable pattern that I can see): https://github.com/hashicorp/boundary/blob/main/internal/session/query.go, https://github.com/hashicorp/boundary/blob/main/internal/server/query.go, https://github.com/hashicorp/boundary/blob/main/internal/auth/oidc/query.go. Happy to have concrete suggestions on making it more readable 😅.

Reply:
indenting `on` at the same level as the `inn`n` so the reader can easily gronk their dependencies.   The `dataKeyVersionIdScopeIdQuery` is a great example of lack of readability related to this sort of indenting for dependent clauses.   Also just putting `on` on it's own line makes it harder to read and it would be better to have the `on jr.key_id=j.key_id` on the same line.. if there's an `and` for the `on` clause then a newline seems warranted.   In general putting very SQL keyword on it's own line and aligning their indentation without out regard to dependencies and operands isn't great for readability IMO.

Reply:
I reindented this to put the `on` indented relative to the `join`.